### PR TITLE
Add protocol scheme to URL of Mesos Master

### PR DESCRIPTION
**Pull request by wndhydrnt**
This PR adds the protocol scheme to the URL used to query the Mesos
Master for new Mesos Slaves if it is not already set in the Location
header.

### CODE REVIEW

**Comment by wndhydrnt** [comment]
Starting from 0.23.0, a Mesos Master [does not](https://github.com/apache/mesos/blob/4ce5475346a0abb7ef4b7ffc9836c5836d7c7a66/src/master/http.cpp#L439)
set the protocol scheme in the redirect URL.

File: main.go [hunk 1]
```diff
@@ -274,8 +285,16 @@ func (e *periodicExporter) updateSlaves() {
 
 	log.Debugf("current elected master at: %s", masterLoc)
 
+	// Starting from 0.23.0, a Mesos Master does not set the scheme in the "Location" header
```
Comment (juliusv) [inline main.go]: Periods after comment sentences please.

**Comment by tommyulfsparre** [comment]
LGTM!

**Comment by antonlindstrom** [comment]
LGTM! Thanks @wndhydrnt! :+1:

**Comment by fabxc** [comment]
Thanks!

**Pull request merged**